muziekweb_api/__init__.py

```diff
@@ -36,3 +37,33 @@ def get_album_information(key: str):
             return minidom.parseString(body)
 
     return None
+
+def get_track_information(key: str):
+    global _api_activated
+
+    if _api_activated:
```

Comment:
if you're only reading this value (not setting it) then you shouldn't need to set `_api_activated` at the beginning of the method.

Reply:
removed